package.json

```diff
@@ -35,56 +35,56 @@
     ]
   },
   "dependencies": {
-    "@angular/animations": "~13.0.0",
-    "@angular/common": "~13.0.0",
-    "@angular/compiler": "~13.0.0",
-    "@angular/core": "~13.0.0",
-    "@angular/forms": "~13.0.0",
-    "@angular/localize": "^13.0.0",
-    "@angular/platform-browser": "~13.0.0",
-    "@angular/platform-browser-dynamic": "~13.0.0",
-    "@angular/router": "~13.0.0",
+    "@angular/animations": "^14.0.0",
+    "@angular/common": "^14.0.0",
+    "@angular/compiler": "^14.0.0",
+    "@angular/core": "^14.0.0",
+    "@angular/forms": "^14.0.0",
+    "@angular/localize": "^14.0.0",
+    "@angular/platform-browser": "^14.0.0",
+    "@angular/platform-browser-dynamic": "^14.0.0",
+    "@angular/router": "^14.0.0",
     "@ng-bootstrap/ng-bootstrap": "^9.0.2",
     "bootstrap": "^4.5.0",
     "rxjs": "~6.5.5",
     "tslib": "^2.3.1",
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~13.0.1",
-    "@angular-eslint/builder": "13.0.1",
-    "@angular-eslint/eslint-plugin": "13.0.1",
-    "@angular-eslint/eslint-plugin-template": "13.0.1",
-    "@angular-eslint/schematics": "^13.0.1",
-    "@angular-eslint/template-parser": "13.0.1",
-    "@angular/cli": "~13.0.1",
-    "@angular/compiler-cli": "~13.0.0",
-    "@angular/language-service": "~13.0.0",
-    "@semantic-release/changelog": "^3.0.4",
-    "@stackblitz/sdk": "^1.3.0",
-    "@types/jasmine": "~3.6.0",
-    "@types/jasminewd2": "^2.0.6",
-    "@types/node": "^12.11.1",
-    "@typescript-eslint/eslint-plugin": "4.3.0",
-    "@typescript-eslint/parser": "4.3.0",
-    "coveralls": "^3.0.2",
-    "eslint": "7.32.0",
-    "eslint-plugin-import": "^2.25.2",
-    "eslint-plugin-jsdoc": "30.7.6",
+    "@angular-devkit/build-angular": "^14.0.0",
+    "@angular-eslint/builder": "13.2.1",
```

Comment:
I think you can use `@angular-eslint/schematics@next` (see: https://github.com/angular-eslint/angular-eslint/pull/1004)

Reply:
I updated the package, still same error. I think this is new rule by added by angular team and inherited by angular-eslint. So the best option would be rename the `@Output` variables to match the aliases. This would fix all errors.

Reply:
I actually didn't mean updating this would fix the lint errors you were referring to just that this could be bumped to v14 too :sweat_smile: 

Because `blur()` already exists you're right there's not much you can do about it.

It's just a lint rule and it can be disabled, but you may want to just disable it in the source for the specific line in question `// eslint-disable-next-line no-output-native, no-output-rename`. The rule `no-output-native` sounds a little more important, but potentially double triggering an event probably isn't that big of a deal and it's already happening.

By disabling it for only the lines in question you'll leave the lint rule on for any new code that's added so that future changes will not add more potential issues.

Reply:
I assume this comment was meant for https://github.com/ng-select/ng-select/pull/2084#discussion_r890254327 but yes, you if you supply a default that will be used as part of what is using for computing the type. If you _need_ to specify the type for some other reason you could use `FormControl<type>` but it is not strictly needed.